README.md

```diff
@@ -196,7 +196,11 @@ The following resources include cisco types and providers along with cisco provi
   * [`cisco_tacacs_server`](#type-cisco_tacacs_server)
   * [`tacacs_server_group (netdev_stdlib)`](#type-cisco_tacacs_server_group)
   * [`cisco_tacacs_server_host`](#type-cisco_tacacs_server_host)
+<<<<<<< HEAD
```

Comment:
@robert-w-gries we're not adding these nedev demos to demo_all. Please speak to @mikewiebe on future plans to come in another PR.

I'll fix the merge conflicts now